### PR TITLE
Shadow updates based on Sketch library

### PR DESCRIPTION
**Pull request by julio8a**
shadow 1 and 2 updates.

### CODE REVIEW

File: dist/variables.css [hunk 1]
```diff
@@ -20,10 +20,10 @@
...
   --code-tracking: 0; 
   --code-family: 'Consolas','Andale Mono','Lucida Console','Monaco',monospace; 
-  --shadow-1: 0 0 12px 0 rgba(0,0,0,0.05); 
-  --shadow-1--hover: 0 0 16px 0 rgba(0,0,0,0.1); 
-  --shadow-1--press: 0 0 16px 0 rgba(0,0,0,0.2); 
-  --shadow-2: 0 0 16px 0 rgba(0,0,0,0.15); 
-  --shadow-2--hover: 0 0 20px 0 rgba(0,0,0,0.2); 
-  --shadow-2--press: 0 0 20px 0 rgba(0,0,0,0.3); 
+  --shadow-1: 0 0 8px 0 rgba(0,0,0,.08);
+  --shadow-1:hover: 0 0 16px 0 rgba(0,0,0,.08);
```
Comment (macandcheese) [inline dist/variables.css]: Should be `--shadow-1--hover`, same for below.

Comment (julio8a) [reply dist/variables.css]: I'll update them all to hover/press since it keeps the naming shorter. I will update them in calcite-components too.

Comment (julio8a) [reply dist/variables.css]: Updated

**Comment by julio8a** [comment]
Okay, take another look.

**Comment by julio8a** [comment]
@macandcheese, ready for another review.

**Comment by macandcheese** [comment]
LGTM

**Pull request merged**